<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Jeu_Carcassonne</class>
    <widget class="QDialog" name="Jeu_Carcassonne">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>756</width>
                <height>508</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <layout class="QVBoxLayout" name="verticalLayout_3">
                                    <item>
                                        <widget class="QLabel" name="label_16">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="text">
                                                <string>Pioche</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QProgressBar" name="progressBar">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="value">
                                                <number>24</number>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <layout class="QVBoxLayout" name="verticalLayout_5">
                                            <item>
                                                <layout class="QVBoxLayout" name="verticalLayout_2">
                                                    <item>
                                                        <widget class="QLabel" name="label_2">
                                                            <property name="sizePolicy">
                                                                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                                                    <horstretch>0</horstretch>
                                                                    <verstretch>0</verstretch>
                                                                </sizepolicy>
                                                            </property>
                                                            <property name="text">
                                                                <string>Ins√©rer Points</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                    <item>
                                                        <widget class="QLabel" name="label_3">
                                                            <property name="sizePolicy">
                                                                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                                                    <horstretch>0</horstretch>
                                                                    <verstretch>0</verstretch>
                                                                </sizepolicy>
                                                            </property>
                                                            <property name="text">
                                                                <string>Attendre fin tour</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                    <item>
                                                        <widget class="QLineEdit" name="lineEdit">
                                                            <property name="sizePolicy">
                                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                                    <horstretch>0</horstretch>
                                                                    <verstretch>0</verstretch>
                                                                </sizepolicy>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                    <item>
                                                        <widget class="QPushButton" name="pushButton_2">
                                                            <property name="text">
                                                                <string>Suivant</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                    <item>
                                                        <spacer name="verticalSpacer_2">
                                                            <property name="orientation">
                                                                <enum>Qt::Vertical</enum>
                                                            </property>
                                                            <property name="sizeHint" stdset="0">
                                                                <size>
                                                                    <width>20</width>
                                                                    <height>40</height>
                                                                </size>
                                                            </property>
                                                        </spacer>
                                                    </item>
                                                </layout>
                                            </item>
                                            <item>
                                                <widget class="QLabel" name="label_tuile">
                                                    <property name="sizePolicy">
                                                        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                                                            <horstretch>0</horstretch>
                                                            <verstretch>0</verstretch>
                                                        </sizepolicy>
                                                    </property>
                                                    <property name="text">
                                                        <string>Image</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QPushButton" name="pushButton">
                                                    <property name="text">
                                                        <string>Tourner</string>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_4">
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout">
                                    <item>
                                        <layout class="QVBoxLayout" name="verticalLayout_6">
                                            <item>
                                                <widget class="QLabel" name="label_4">
                                                    <property name="text">
                                                        <string>Joueur1</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QLabel" name="label_5">
                                                    <property name="text">
                                                        <string>Infos Ressources</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QLabel" name="label_6">
                                                    <property name="text">
                                                        <string>Score : 0</string>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <layout class="QVBoxLayout" name="verticalLayout_8">
                                            <item>
                                                <widget class="QLabel" name="label_10">
                                                    <property name="text">
                                                        <string>Joueur2</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QLabel" name="label_11">
                                                    <property name="text">
                                                        <string>Infos Ressources</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QLabel" name="label_12">
                                                    <property name="text">
                                                        <string>Score : 0</string>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <layout class="QVBoxLayout" name="verticalLayout_9">
                                            <item>
                                                <widget class="QLabel" name="label_13">
                                                    <property name="text">
                                                        <string>Joueur3</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QLabel" name="label_14">
                                                    <property name="text">
                                                        <string>Infos Ressources</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QLabel" name="label_15">
                                                    <property name="text">
                                                        <string>Score : 0</string>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <layout class="QVBoxLayout" name="verticalLayout_7">
                                            <item>
                                                <widget class="QLabel" name="label_7">
                                                    <property name="text">
                                                        <string>Joueur4</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QLabel" name="label_9">
                                                    <property name="text">
                                                        <string>Infos Ressources</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QLabel" name="label_8">
                                                    <property name="text">
                                                        <string>Score : 0</string>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="100,0">
                                    <item>
                                        <widget class="QScrollArea" name="scrollArea">
                                            <property name="widgetResizable">
                                                <bool>true</bool>
                                            </property>
                                            <widget class="QWidget" name="scrollAreaWidgetContents">
                                                <property name="geometry">
                                                    <rect>
                                                        <x>0</x>
                                                        <y>0</y>
                                                        <width>367</width>
                                                        <height>425</height>
                                                    </rect>
                                                </property>
                                            </widget>
                                        </widget>
                                    </item>
                                    <item>
                                        <layout class="QVBoxLayout" name="verticalLayout_10">
                                            <item>
                                                <widget class="QListWidget" name="listWidget"/>
                                            </item>
                                            <item>
                                                <layout class="QHBoxLayout" name="horizontalLayout_5">
                                                    <item>
                                                        <widget class="QPushButton" name="pushButton_5">
                                                            <property name="text">
                                                                <string>OK</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                    <item>
                                                        <widget class="QPushButton" name="pushButton_4">
                                                            <property name="text">
                                                                <string>Annuler</string>
                                                            </property>
                                                        </widget>
                                                    </item>
                                                </layout>
                                            </item>
                                        </layout>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
